flang/lib/Optimizer/Transforms/RewriteLoop.cpp

```diff
@@ -292,12 +293,10 @@ class CfgIterWhileConv : public mlir::OpRewritePattern<fir::IterWhileOp> {
 class CfgConversion : public CFGConversionBase<CfgConversion> {
 public:
   void runOnFunction() override {
-    if (disableCfgConversion)
```

Comment:
Can you add this to the set of "driver" options?

Reply:
Option added.